src/arch/bbcmicro/bbcmicro.ld

```diff
@@ -1,6 +1,6 @@
 MEMORY {
     zp : ORIGIN = 0, LENGTH = 0x90
-    ram (rw) : ORIGIN = 0x400, LENGTH = 0x400
+    ram (rw) : ORIGIN = 0x400, LENGTH = 0x800
```

Comment:
This doesn't work, unfortunately --- the application workspace at 0x400 is only 1kB long, and is followed by the sound workspace, so loading this on a stock BBC Micro will have some very odd effects (see https://www.youtube.com/watch?v=4icSz0ks2pE for when I found this out!).

I was actually doing some fiddling with this myself and I think the bulk of the new serial code in biosentry is unused, and can probably be dropped. Let me take a look.

Reply:
Ah, that probably explains the continuous beep I heard with b-em :) I thought it had something to do with my emulator settings, but I didn't here it before.